raiden/utils/capabilities.py

```diff
@@ -59,7 +59,12 @@ def capconfig_to_dict(config: CapabilitiesConfig) -> Dict[str, Any]:
     result[Capabilities.NO_MEDIATE.value] = config.no_mediate
     result[Capabilities.NO_DELIVERY.value] = config.no_delivery
     result[Capabilities.WEBRTC.value] = config.web_rtc
-    for key in config.__dict__.keys():
+    other_keys = [
+        key
+        for key in config.__dict__.keys()
+        if key not in ["no_receive", "no_mediate", "no_delivery", "web_rtc"]
```

Comment:
This could stay generic/forward compatible:
```suggestion
        if key not in CapabilitiesConfig.__annotations__.keys()
```

Reply:
True, but only for the part of filtering. The function itself still remains non generic due to the above assignments. I would like to make it generic in a following PR